src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -4121,7 +4123,7 @@ private bool ScanXmlCrefToken(ref TokenInfo info)
 
                 TextWindow.Reset(beforeConsumed);
 
-                if (this.ScanIdentifier(ref info) && info.Text.Length > 0)
+                if (this.ScanIdentifier(ref info) && info.Text!.Length > 0)
```

Comment:
It looks like there are several situations in this file where dotnet/csharplang#4563 would be useful.